answers/views.py

```diff
@@ -35,8 +35,10 @@ class AnswerView(LoginRequiredMixin, View):
     def get(self, request, hunt_pk):
         hunt = get_object_or_404(Hunt, pk=hunt_pk)
         answers = Answer.objects.filter(puzzle__hunt__pk=hunt_pk).order_by('-created_on')
-        status_forms = [UpdateAnswerStatusForm(initial={'status': ans.get_status()}) for ans in answers]
-        notes_forms = [UpdateAnswerNotesForm(initial={'text': ans.get_notes()}) for ans in answers]
+        status_forms = [UpdateAnswerStatusForm(initial={'status': ans.get_status()},
+                                               auto_id=False) for ans in answers]
```

Comment:
I see you added `auto_id=False` in a lot of places. Does it make a significant difference in performance?

Reply:
No difference in performance; just cleans up the messages in the console.

Reply:
Oh nice, thanks for fixing those warnings!